crawler/constants.py

```diff
@@ -37,9 +49,36 @@
 MLWH_DATE_TESTED = "date_tested"
 MLWH_SOURCE = "source"
 MLWH_LAB_ID = "lab_id"
+MLWH_CH1_TARGET = "ch1_target"
+MLWH_CH1_RESULT = "ch1_result"
+MLWH_CH1_CQ = "ch1_cq"
+MLWH_CH2_TARGET = "ch2_target"
+MLWH_CH2_RESULT = "ch2_result"
+MLWH_CH2_CQ = "ch2_cq"
+MLWH_CH3_TARGET = "ch3_target"
+MLWH_CH3_RESULT = "ch3_result"
+MLWH_CH3_CQ = "ch3_cq"
+MLWH_CH4_TARGET = "ch4_target"
+MLWH_CH4_RESULT = "ch4_result"
+MLWH_CH4_CQ = "ch4_cq"
 MLWH_CREATED_AT = "created_at"
 MLWH_UPDATED_AT = "updated_at"
 
 # datetime formats
 MONGO_DATETIME_FORMAT = "%y%m%d_%H%M"
 MYSQL_DATETIME_FORMAT = "%Y-%m-%d %H:%M:%S"
+
+# allowed Result field values
+# TODO: confirm! 'Detected', 'Not detected'?
+ALLOWED_RESULT_VALUES = ('Positive', 'Negative', 'limit of detection', 'Void')
+
+# allowed CT channel CHn-Target field values (or can be null)
+ALLOWED_CH_TARGET_VALUES = ('ORF1ab', 'N gene', 'S gene', 'MS2')
+
+# allowed CT channel CHn-Result field values (or can be null)
+ALLOWED_CH_RESULT_VALUES = ('Positive', 'Negative', 'Inconclusive', 'Void')
+
+# range of allowed cq values (0 .. 100, set as strings for conversion to decimals in code)
+MIN_CQ_VALUE = '0.0'
+MAX_CQ_VALUE = '100.0'
```

Comment:
Small thing, but I think I would personally store them as decimals here:
e.g. `MIN_CQ_VALUE = Decimal('0.0')`
and then make the `is_within_cq_range` method expect 3 decimals

Reply:
Changed.